packages/vite-plugin-cloudflare/src/miniflare-options.ts

```diff
@@ -189,6 +190,43 @@ function getEntryWorkerConfig(
 	];
 }
 
+function filterTails(
+	tails: WorkerOptions["tails"],
+	userWorkers: { name?: string }[],
+	log: (msg: string) => void
+) {
+	// Only connect the tail consumers that represent Workers that are defined in the Vite config. Warn that a tail will be omitted otherwise
+	// This _differs from service bindings_ because tail consumers are "optional" in a sense, and shouldn't affect the runtime behaviour of a Worker
+	return tails?.filter((tailService) => {
+		let name: string;
+		if (typeof tailService === "string") {
+			name = tailService;
+		} else if (
+			typeof tailService === "object" &&
+			"name" in tailService &&
+			typeof tailService.name === "string"
+		) {
+			name = tailService.name;
+		} else {
+			// Don't interfere with network-based tail connections (e.g. via the dev registry), or kCurrentWorker
+			return true;
+		}
+		const found = !!userWorkers.find((w) => w.name === name);
```

Comment:
```suggestion
		const found = userWorkers.some(w => w.name === name);
```

Reply:
Sorry, my bad. I think the checks are failing because I put `w` instead of `(w)`.